hw2/01-coins.js

```diff
@@ -1,7 +1,131 @@
-/** Exercise 01 - Coins **/
+/* eslint-disable no-console */
+/** Exercise 01 - Coins * */
 
+/**
+ * Removes as many dollars as possible and returns
+ * amount removed and amount remaing
+ */
+const removeDollar = (num) => {
```

Comment:
I like the code readability and reusability by separating each coin functionality into their own function.